### ISSUE TITLE
Warnings when building with latest docker for mac

### ISSUE THREAD

**Issue by bootstraponline** [bitrise-docker/android#117]
```
[WARNING]: Empty continuation line found in:
    RUN sdkmanager     "platforms;android-26"     "platforms;android-25"     "platforms;android-24"     "platforms;android-23"     "platforms;android-22"     "platforms;android-21"     "platforms;android-19"     "platforms;android-17"     "platforms;android-15"     "platforms;android-10"     "build-tools;26.0.1"     "build-tools;25.0.3"     "build-tools;24.0.3"     "build-tools;23.0.3"     "build-tools;22.0.1"     "build-tools;21.1.2"     "build-tools;19.1.0"     "build-tools;17.0.0"     "system-images;android-25;google_apis;armeabi-v7a"     "system-images;android-24;default;armeabi-v7a"     "system-images;android-22;default;armeabi-v7a"     "system-images;android-21;default;armeabi-v7a"     "system-images;android-19;default;armeabi-v7a"     "extras;android;m2repository"     "extras;google;m2repository"     "extras;google;google_play_services"     "extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2"     "extras;m2repository;com;android;support;constraint;constraint-layout;1.0.1"     "add-ons;addon-google_apis-google-23"     "add-ons;addon-google_apis-google-22"     "add-ons;addon-google_apis-google-21"
[WARNING]: Empty continuation lines will become errors in a future release.
```

There are a bunch more warnings. Not sure which ones are legit or not.

**Comment by viktorbenei** [comment]
Thanks @bootstraponline , good catch! ;)

I scheduled a check / fix for this (ping @ghaabor )

**Comment by ghaabor** [comment]
Thanks for the report @bootstraponline, we've fixed the Dockerfile in #126.

**Comment by bootstraponline** [comment]
Nice. Thanks
